eva-lib/src/test/java/uk/ac/ebi/eva/lib/metadata/StudyEvaproDBAdaptorTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2014-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.metadata;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.opencb.datastore.core.QueryOptions;
+import org.opencb.datastore.core.QueryResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.ac.ebi.eva.lib.models.VariantStudy;
+import uk.ac.ebi.eva.lib.utils.QueryOptionsConstants;
+
+import java.util.function.Predicate;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class StudyEvaproDBAdaptorTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Autowired
+    private StudyEvaproDBAdaptor studyEvaproDBAdaptor;
+
+    @Before
+    public void setUp() throws Exception {
+        EvaStudyBrowserTestData.persistTestData(entityManager);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        entityManager.clear();
+    }
```

Comment:
Isn't it enough to make these `@BeforeClass` and `@AfterClass` methods? The class being tested only has read operations, so the data shouldn't change between test cases.

I don't know how costly is to create an in-memory `EntityManager`, but if they are remotely similar to the regular ones this could help to speed up the tests execution time.

Applies to other test classes.